editor/plugins/org.fusesource.ide.wsdl2rest.ui/META-INF/MANIFEST.MF

```diff
@@ -39,9 +39,10 @@ Bundle-ClassPath: .,
  libs/commons-beanutils.jar,
  libs/commons-collections.jar,
  libs/commons-digester.jar,
- libs/commons-lang.jar,
  libs/commons-logging.jar,
  libs/commons-validator.jar,
+ libs/commons-lang.jar,
```

Comment:
I think that it is working when launching test locally as PDE test because the commons-lang has been kept here but is missing from build.properties and/or .classpath